From randy.dunlap@oracle.com  Fri Aug  7 13:34:00 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Fri, 7 Aug 2009 11:17:04 -0700
Subject: Staging: hv: blk dev depends on SCSI
To: Stephen Rothwell <sfr@canb.auug.org.au>, gregkh@suse.de, akpm <akpm@linux-foundation.org>
Message-ID: <20090807111704.b7b79983.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

hv block driver uses scsi_*() interfaces so it should depend on SCSI.

ERROR: "scsi_normalize_sense" [drivers/staging/hv/hv_blkvsc.ko] undefined!
ERROR: "scsi_print_sense_hdr" [drivers/staging/hv/hv_blkvsc.ko] undefined!

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/hv/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/hv/Kconfig
+++ b/drivers/staging/hv/Kconfig
@@ -15,7 +15,7 @@ config HYPERV_STORAGE
 
 config HYPERV_BLOCK
 	tristate "Microsoft Hyper-V virtual block driver"
-	depends on BLOCK && m
+	depends on BLOCK && SCSI && m
 	default n
 	help
 	  Select this option to enable the Hyper-V virtual block driver.
